M/Minuit2/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Minuit2"
+version = v"6.15.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/jstrube/Minuit2.git", "32b4337fab6aca8cde5a0e18dfd9e05130eb7184"),
```

Comment:
What's the upstream source?

Reply:
Minuit2 is part of the ROOT distribution, but there's no separate package. There's a CMake file in the minuit dir of the ROOT source tree that let's you build a self-contained library without pulling in the rest of ROOT. I just put that dir into its own repo to make easier to get the code if you don't want the rest of ROOT.

Reply:
So if I wanted to submit a patch upstream where I should look?

Reply:
I had to look, and amazingly, it's actually on GitHub now... https://github.com/root-project/root. One could probably change the build script to download the whole tree and still only build minuit2. That would be easier for keeping things in sync.